src/fix_wall_region.cpp

```diff
@@ -28,15 +28,15 @@ using namespace LAMMPS_NS;
 using namespace FixConst;
 using namespace MathConst;
 
-enum{LJ93,LJ126,LJ1043,COLLOID,HARMONIC};
+enum{LJ93,LJ126,LJ1043,MORSE,COLLOID,HARMONIC};
```

Comment:
Please always add new items to enumerators at the back. That minimizes the chance for unexpected behavior.

Reply:
Ok, no problem, I just felt that this would be a natural position for Morse in this ordering, after the L-Js... Let's say in the complexity/similarity of the potentials or something like that.